subprojects/docs/src/samples/signing/gnupg-signatory/build.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'java'
+apply plugin: 'signing'
+
+group = 'gradle'
+version = '1.0'
+
+// Typically set in ~/.gradle/gradle.properties
+project['signing.gnupg.executable'] = null
+project['signing.gnupg.useLegacyGpg'] = "true"
+project['signing.gnupg.homeDir'] = "gnupg-home"
+project['signing.gnupg.optionsFile'] = null
+project['signing.gnupg.keyName'] = "24875D73"
+project['signing.gnupg.passphrase'] = "gradle"
+
+// START SNIPPET configure-signatory
+import org.gradle.plugins.signing.signatory.gnupg.GnupgSignatoryProvider
+
+signing {
+    signatories = new GnupgSignatoryProvider()
```

Comment:
As a side note: I know that the `signatories` already exists. To me this indicates that there are possibly multiple signatories. However, you can only assign a single signatory provider. The naming seems off.

Reply:
I find the naming confusing, too. The plural seems to be chosen, though, because it is not the actual signatory that is set, but a factory for signatories. As far as I understand, it is technically possible, to configure multiple named signatory instances. However, I have not found any documentation yet that explains how to actually do this.

Since there is also a `getSignator()` method in `SigningExtension` which returns the default signatory, the property name cannot simply be changed to `signatory`.

Reply:
I find the naming confusing, too. The plural seems to be chosen, though, because it is not the actual signatory that is set, but a factory for signatories. As far as I understand, it is technically possible, to configure multiple named signatory instances. However, I have not found any documentation yet that explains how to actually do this.

Since there is also a `getSignatory()` method in `SigningExtension` which returns the default signatory, the property name cannot simply be changed to `signatory`.